python-refactor/python-refactor.yaml

```
@@ -0,0 +1,23 @@
+name: python-refactor
+description: Python 2 to 3.
+
+# Python 2 to 3.
+
+on:
+  - repositoriesMatchingQuery: lang:python print ":[string]" -repo:python-language-server -repo:ctags fork:yes patternType:structural 
+
+# In each repository
+steps:
+  - run: |
+      pip install 2to3 
+      2to3 -w .
+    container: python:3.7.5-slim
```

We may as well track the current stable version, at least for now:

```suggestion
    container: python:3-slim
```